### ISSUE TITLE
Chrome Driver 2.18 IllegalStateException

### ISSUE THREAD

**Issue by TheFifthFreedom** [bonigarcia/webdrivermanager#5]
Interestingly, with the new release of Chrome Driver (2.18), the path to the driver executable is not set automatically by `webdrivermanager` anymore. When running:
```java
ChromeDriverManager.getInstance().setup();
driver = new ChromeDriver(options);
```
I run into an IllegalStateException:
```
Exception in thread "main" java.lang.IllegalStateException: The path to the driver executable must be set by the webdriver.chrome.driver system property; for more information, see https://github.com/SeleniumHQ/selenium/wiki/ChromeDriver. The latest version can be downloaded from http://chromedriver.storage.googleapis.com/index.html
	at com.google.common.base.Preconditions.checkState(Preconditions.java:197)
	at org.openqa.selenium.remote.service.DriverService.findExecutable(DriverService.java:109)
	at org.openqa.selenium.chrome.ChromeDriverService.access$000(ChromeDriverService.java:32)
	at org.openqa.selenium.chrome.ChromeDriverService$Builder.findDefaultExecutable(ChromeDriverService.java:137)
	at org.openqa.selenium.remote.service.DriverService$Builder.build(DriverService.java:290)
	at org.openqa.selenium.chrome.ChromeDriverService.createDefaultService(ChromeDriverService.java:88)
	at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:148)
	at co.quantent.engine.SeleniumDriver.<init>(SeleniumDriver.java:33)
	at co.quantent.engine.Tester.main(Tester.java:11)
```
However, if I specify I want Chrome Driver 2.17 instead of 2.18, like so:
```java
ChromeDriverManager.getInstance().setup("2.17");
driver = new ChromeDriver(options);
```
Everything runs fine! Would you happen to know what's going on?

**Comment by bonigarcia** [comment]
Version 2.18 of chromedriver has been released today (19th August 2015). The error you experienced is a temporal issue related to this reason.

This afternoon (CEST time) there were only two versions of chromedriver driver: linux32 and linux64. I suppose you experienced the error running a test on Windows or Mac.

At time of writing time this post (Thu Aug 20 00:05:14 CEST 2015) there are three versions of chromedriver 2.18:

![screenshot from 2015-08-19 23 51 35](https://cloud.githubusercontent.com/assets/5413013/9370262/4494e396-46cf-11e5-9682-9c32e9b25b8e.png)

Presumably the chromedriver 2.18 for Windows is released very soon.

Nevertheless, in order to prevent this potential problem, I have improved the logic of webdrivermanager to be smarter when finding out the latest version of a webdriver binary. As of version 1.2.2 (just released), if the latest version of a given platform is not found, the next released is seeked. In your sample, version of chromedriver 2.17 would be downloaded automatically.
